README.md

```diff
@@ -30,249 +29,126 @@ TOC
 
 ### Hardware
 
-To get the most out of your Bridgehead, we recommend the follwing Hardware:
+Hardware requirements strongly depend on the specific use-cases of your network as well as on the data it is going to serve. Most use-cases are well-served with the following configuration:
 
 - 4 CPU cores
-- At least 8 GB Ram
-- 100GB Hard Drive, SSD recommended
-
-### System
-
-You are strongly recommended to install the Bridgehead under a Linux operating system (but see the section [Non-Linux OS](#non-linux-os)). You will need root (administrator) priveleges on this machine in order to perform the deployment.
+- 32 GB RAM
+- 160GB Hard Drive, SSD recommended
```

Comment:
Will 8/16 GB of RAM impede the performance to a substantial degree? Asking if we can make it work with 8 for hospitals with limited resources.

Reply:
I guess we can, try it out and correct the numbers based on your experiences. I'm hard-pressed to give any lower number in the documentation, though.